src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/General/URLValidation.feature

```diff
@@ -339,60 +353,132 @@ Feature: Validation of the structure of the URLs
                       ]
                   }
                   """
-              And the response headers includes
+              And the response headers include
                   """
-                  Content-Type: application/json; charset=utf-8
+                    {
+                        "Content-Type": "application/json; charset=utf-8"
+                    }
                   """
 
-        ## The resolution of this ticket will solve the execution error: https://edfi.atlassian.net/browse/DMS-354
-        @API-078 @ignore
-        Scenario: 12 Ensure clients validate required identifier on DELETE requests
-             When a GET request is made to "/ed-fi/SCHOOLS?OfFSeT=1&LImiT=2&totalCount=TRue"
+        @API-235
+        Scenario: 12 Ensure client can retrieve information through a case insensitive query
+            Given a POST request is made to "/ed-fi/classPeriods" with
+                  """
+                  {
+                      "schoolReference": {
+                          "schoolId": 255901044
+                      },
+                      "classPeriodName": "Class Period Test",
+                      "officialAttendancePeriod": true
+                  }
+                  """
+             When a GET request is made to "/ed-fi/classPeriods?classPeriodName=CLASS+pERIOD+test"
              Then it should respond with 200
               And the response body is
                   """
                   [
                       {
-                          "id": "49fac44c76ad417a9511101df379718f",
-                          "schoolId": 5,
-                          "nameOfInstitution": "UT Austin College of Education Graduate",
-                          "addresses": [
-                              {
-                                  "addressTypeDescriptor": "uri://ed-fi.org/AddressTypeDescriptor#Physical",
-                                  "city": "Austin",
-                                  "postalCode": "78712",
-                                  "stateAbbreviationDescriptor": "uri://ed-fi.org/StateAbbreviationDescriptor#TX",
-                                  "streetNumberName": "1912 Speedway Stop D5000",
-                                  "nameOfCounty": "Travis",
-                                  "periods": []
-                              }
-                          ],
-                          "educationOrganizationCategories": [
-                              {
-                                  "educationOrganizationCategoryDescriptor": "uri://tpdm.ed-fi.org/EducationOrganizationCategoryDescriptor#Educator Preparation Provider"
-                              }
-                          ],
-                          "identificationCodes": [],
-                          "indicators": [],
-                          "institutionTelephones": [],
-                          "internationalAddresses": [],
-                          "_ext": {
-                              "tpdm": {}
```

Comment:
The old response was copied from the ODS/API but doesn't match the school that we created up above. It never would have passed as written.